lntest/harness.go

```diff
@@ -1383,6 +1383,21 @@ func (n *NetworkHarness) sendCoins(ctx context.Context, amt btcutil.Amount,
 	return target.WaitForBalance(expectedBalance, true)
 }
 
+// SlowGenerate generates blocks with a large time interval between them. This
+// is useful for debugging.
+func (n *NetworkHarness) SlowGenerate(nb uint32) ([]*chainhash.Hash, error) {
```

Comment:
So the plan is to remove this method in a final commit? I think we should also remove it all together before this PR is merged, unless we think it's useful for other things.

Reply:
Personally, I'd leave it since it's sometimes useful to quickly test out replacing the regular mining functions for this, but I'll defer to whatever you prefer. I can remove it once this is cleared for merging.

Reply:
If we don't use the methods, the linter will probably complain about them being unused.
On the other hand, maybe it would make sense to actually run the slow mining code in the itests if that surfaces the problem more reliably. Perhaps 3 seconds is a bit extreme and we can use a shorter interval that still surfaces the problem.
How much extra time does this add to the tests? When we can actually run them in parallel with other tests (see #4655) a few extra seconds might not matter much if we have more reliable tests.

Reply:
However many blocks are mined (for the serial case) if you do this on every block. Don't know exact count for lnd but probably upwards of 1000+ blocks.

Reply:
I meant just leaving in the slow mining added by 35e7b7f638e56f36e56c13721a93ad60a00a79a0 and reducing the values in 3feddc0b41345a0033b85ce316f52e2a29f600b7 to 1 second instead of 3. But I guess this still adds up to ~40 seconds (`finalCltvDelta`) to the `local_force_close_on-chain_htlc_timeout` test.
So probably better remove the calls to `slowMineBlocks`. I'm fine with leaving the functions themselves, they'll just need a `//nolint` comment (probably).